governance.md

```diff
@@ -57,6 +57,18 @@ Qualification guidelines
 
 If you’d like to nominate yourself as an Approver, please create an [Issue in the community repository](https://github.com/spinnaker/community/issues). 
 
+### SIG Leads
+
+SIG Leads are responsible for management of a particular Special Interest Group within the project, managing day-to-day operations of their interest scope.
+
+Duties
+
+* Set agenda for, facilitate, and drive SIG meetings
+* Review and sponsor proposals related to the SIG
+* Triage inbound spinnaker/spinnaker issues related to the SIG
+* Review and merge or decline pull requests from the community
```

Comment:
Excuse my ignorance of GitHub workflow, but are Leads who are not Approvers going to be able to do this?

Reply:
No, they won't, which is why having a qualification of being an Approver was part of the original PR. The issue is that there are SIGs that do not have anything to do with the actual code (community, documentation, etc). Perhaps there should be language that Approver is required for Leads who are in SIGs involving the codebase? I could use help with this.

Reply:
I've updated things -- I'm going to merge this PR and we can refine the language in a subsequent edit of the document.